inspire_json_merger/pre_filters.py

```diff
@@ -188,5 +188,28 @@ def _remove_if_falsy(pmap, key):
         return pmap
 
 
+def remove_duplicated_titles(root, head, update):
+    new_elements = []
+    for data in (root, head, update):
+        titles = data.get('titles')
+        if not titles:
+            new_elements.append(data)
+            continue
+        titles_dict = dict()
+        to_delete = []
+        for idx, title in enumerate(titles):
+            if title['title'] not in titles_dict:
+                titles_dict[title['title']] = title
+            else:
+                if title['source'].lower() != 'arxiv':
+                    to_delete.append(titles_dict[title['title']])
```

Comment:
won't this fail if there's more than one non-arXiv title? it shouldn't happen very often, but it's not forbidden in principle.

Reply:
One of them will be always deleted